packages/api/__tests__/index.js

```diff
@@ -292,9 +292,9 @@ describe('/api/profile/createUser', () => {
     });
   });
 
-  it('should return 201 on creation of new user', (done) => {
+  it('should return 200 on creation of new user', (done) => {
     tokenService.generate({
-      email: 'vipulrawat007@gmail.com',
+      email: 'vipultests@gmail.com',
```

Comment:
If you want to send these test emails to a dummy email address, use a public-inbox service like mailinator. No need to go through the hassle of creating a new gmail account